coalib/bears/requirements/DistributionRequirement.py

```diff
@@ -1,4 +1,9 @@
-import platform
+import subprocess
+
+try:
+    import distro
+except subprocess.CalledProcessError:  # pragma: no cover
+    pass
```

Comment:
if I do this, the rest of the functionality would break with windows, need suggestions.